pkg/controller/windowsmachineconfig/windows/service.go

```diff
@@ -0,0 +1,46 @@
+package windows
+
+// service represents a Windows service
+type service interface {
+	getName() string
+	getBinaryPath() string
+	getArgs() string
```

Comment:
As per go naming conventions / suggestions I don't think you need the `get` prefix.

Reply:
if the methods are not exported an issue occurs where you cant have a function and a struct field with the same name